app/src/main/java/com/teamagam/gimelgimel/app/message/viewModel/adapter/MessagesRecyclerViewAdapter.java

```diff
@@ -164,6 +170,12 @@ private boolean isAlertMessage(MessageApp messageApp) {
         @BindView(recycler_message_listitem_layout)
         RelativeLayout container;
 
+        @BindView(R.id.image_viewholder_layout)
+        FrameLayout imageViewHolder;
```

Comment:
viewHolder is confusing.
Its's and overload of something else that is used right here.
I'd rename it to imageContainerLayout or anything else really.